.github/workflows/PR.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
       - name: Run ubuntu tests
         if: matrix.os == 'ubuntu-latest'
-        run: ./gradlew --stacktrace
+        run: ./gradlew build --stacktrace
```

Comment:
whoooops. Gradle should really fail if you do this.

Reply:
ya big woops. luckily things were green